scripts/ci/analyze-commits.js

```diff
@@ -16,15 +16,19 @@ var blocked_statements = [
 ];
```

Comment:
Sure, will change that.
By the way, now it works perfectly, just tested it:

![image](https://cloud.githubusercontent.com/assets/4987015/13927806/806e1140-ef93-11e5-8dcf-2701512d9054.png)

cc. @hansl